server/routerlicious/packages/routerlicious-base/src/riddler/tenantManager.ts

```diff
@@ -289,11 +301,13 @@ export class TenantManager {
 							},
 						);
 					}
+					this.logInvalidTokenJti(tenantId, token, error.code);
```

Comment:
nit: could combine both of these logInvalid calls in this catch to be just below `isNetworkError` check.